includes/db.inc

```diff
@@ -296,10 +297,12 @@ function islandora_solr_get_collection_sort_string($pid, $use_fallbacks = FALSE)
   if ($use_fallbacks) {
     $sort = variable_get('islandora_solr_collection_sort', '');
     if (!empty($sort)) {
+      $sort = islandora_solr_set_default_collection_sort_order($sort);
```

Comment:
If you are going to alter the default strings, then I think this should be done when they are saved so use of them would be consistent by all of Islandora.